qualang_tools/wirer/connectivity/wiring_spec.py

```diff
@@ -34,13 +34,17 @@ class WiringLineType(Enum):
     FLUX = "z"
     CHARGE = "q"
     COUPLER = "c"
+    CROSS_DRIVE = "xd"
```

Comment:
This could be simply `CROSS_RESONANCE = "cr"` as we are adding `ZZ_DRIVE` as well? (I called it cross_drive to keep it applicable both to cross resonance and zz drive). I think I'm wondering if adding new enums as needed in the future would be what it is meant here.

Reply:
Thanks, "cr" is much cleaner than "xd".

Yes, for now I think adding new enums for new drives is useful as it gives then a standard wiring name, e.g. `q1-2.cr`, allowing it to be loaded and referenced later by QuAM in a standard way.